src/Abstractions/NexusMods.Abstractions.Library/ILibraryService.cs

```diff
@@ -28,5 +28,6 @@ public interface ILibraryService
     /// </summary>
     /// <param name="libraryItem">The item to install.</param>
     /// <param name="targetLoadout">The target loadout.</param>
-    IJob InstallItem(LibraryItem.ReadOnly libraryItem, Loadout.ReadOnly targetLoadout);
+    /// <param name="installer">If set, must be set to a ILibraryItemInstaller (untyped here as to not require a library reference). The Library will use this installer to install the item</param>
+    IJob InstallItem(LibraryItem.ReadOnly libraryItem, Loadout.ReadOnly targetLoadout, object? installer = null);
```

Comment:
What's the issue with using `ILibraryItemInstaller`? I changed this on my branch and couldn't find an issue. The `Library*` abstraction projects were designed to prevent dependency issues by splitting up `ILibraryService`, the installers, and the models into separate projects, so there shouldn't really be any issues here.